### DISCUSSION TITLE
文件东西都更新了 为什么网站总是断连 又好 又断开

### DISCUSSION THREAD

**Discussion by SKTBaker** [XIU2/CloudflareSpeedTest#330] [讨论]
有时候死活连不上

**Comment by XIU2** [comment]
墙 / 运营商的问题（因为涉及不同地区不同运营商，因此我更倾向于是**墙**干的）。
你问我没用，这不是我们能解决的事情，或许要考虑放弃 Cloudflare CDN 了，这也是预料之中的事情。
#217

**Comment by SKTBaker** [comment]
我想重装请问怎么删除卸载清空原来的东西呢
重装对连接有影响吗

**Comment by XIU2** [comment]
啥玩意？你来错地方了吧。。。

你说的话，我完全无法理解和该项目有什么关联。。。

CloudflareST 是单文件程序，绿色无依赖的，解压即用，不用就直接删除，压根不存在安装、卸载、重装的说法。

**Comment by SKTBaker** [comment]
> 啥玩意?你来错地方了吧……
> 
> 你说的话,我完全无法理解和该项目有什么关联…
> 
> cloudflare是单文件程序,绿色无依赖的,解压即用,不用就直接删除,压根不存在安装、卸载,重装的说法

> 啥玩意?你来错地方了吧……
> 
> 你说的话,我完全无法理解和该项目有什么关联…
> 
> cloudflare是单文件程序,绿色无依赖的,解压即用,不用就直接删除,压根不存在安装、卸载,重装的说法。

https://www.likecs.com/show-202080.html
我是别的网站到这里的还有知乎 找到了
